db/re/mob_race2_db.txt

```diff
@@ -51,3 +51,6 @@ RC2_ROCKRIDGE,3736,3737,3738,3739,3740,3741,3742,3743,3744,3745,3746,3747,3748,3
 //RC2_HEARTHUNTER,3626,3627,3628,3629
 // Werner Laboratory
 //RC2_WERNER_LAB,3621,3622,3631,3632,3633 // miss ID 3826 - YSF Seyren monster
+// Green Aura 90% Reduced damage
+// Just uncomment the line below to enable reduced damage. PS: The green aura effect is clientside. 
```

Comment:
This comment is useless now. Also, don't forget to uncomment the group below since the battle_config manages this now.